yarn.lock

```diff
@@ -1433,4 +1433,4 @@ yallist@^4.0.0:
 yaml@^2.1.1:
   version "2.1.1"
   resolved "https://registry.yarnpkg.com/yaml/-/yaml-2.1.1.tgz#1e06fb4ca46e60d9da07e4f786ea370ed3c3cfec"
-  integrity sha512-o96x3OPo8GjWeSLF+wOAbrPfhFOGY0W00GNaxCDv+9hkcDJEnev1yh8S7pgHF0ik6zc8sQLuL8hjHjJULZp8bw==
+  integrity sha512-o96x3OPo8GjWeSLF+wOAbrPfhFOGY0W00GNaxCDv+9hkcDJEnev1yh8S7pgHF0ik6zc8sQLuL8hjHjJULZp8bw==
```

Comment:
i don't think we need this change either

Reply:
I'm not sure what's the issue here, I copy pasted the lock file from project root to get rid of issues

Reply:
i believe there's a difference in the trailing endfile character, which you should be able to reset from git

Reply:
reset this file using git, should be fixed now